helm_deploy/values-dev.yaml

```diff
@@ -6,6 +6,12 @@ generic-service:
 
   ingress:
     host: calculate-release-dates-dev.hmpps.service.justice.gov.uk
+    modsecurity_enabled: true
```

Comment:
the below modsecurity_snippet is the same as the generic service template version, except for `DetectionOnly`